lib/model.js

```diff
@@ -3375,6 +3375,7 @@ function _setIsNew(doc, val) {
  * trip to MongoDB.
  *
  * Mongoose will perform casting on all operations you provide.
+ * The only exception is if you pass [set the `update` operator for `updateOne` or `updateMany` to a pipeline](https://www.mongodb.com/docs/manual/reference/method/db.collection.bulkWrite/#updateone-and-updatemany): Mongoose does **not** cast update pipelines.
```

Comment:
`if you pass set the` feels somewhat out-of-place for this sentence, maybe consider rewording